src/plugin/regions/region.js

```diff
@@ -614,10 +616,19 @@ export class Region {
      * @param {string} direction 'start 'or 'end'
      */
     onResize(delta, direction) {
+        let minLength = null;
+
```

Comment:
Maybe add a comment so people know a region's minLength is used over generalMinLength if both are specified

Reply:
ping @marizuccara